### PR TITLE
Updated docs for new crypto builtins

### PR DESCRIPTION
**Pull request by AHaliq**
builtins from PRs
- https://github.com/Zilliqa/scilla/pull/1026
- https://github.com/Zilliqa/scilla/pull/1025

### CODE REVIEW

File: docs/source/scilla-in-depth.rst
Comment (anton-trunov) [inline docs/source/scilla-in-depth.rst:1084]: Let's tweak this description a bit to match the style of the other option-valued builtins. For instance, just above this paragraph it says:

> The return type is ``Option String``. On success, ``Some addr`` is returned and on invalid inputs ``None`` is returned.

File: docs/source/scilla-in-depth.rst
Comment (anton-trunov) [inline docs/source/scilla-in-depth.rst:1096]:
```suggestion
- ``builtin alt_bn128_G1_bmul s``. It is known as basepoint multiplication; the builtin takes a scalar ``ByStr32`` value ``s``
  and uses the generator of the curve to perform multiplication. The result is an option of a point on the curve.
```

File: docs/source/scilla-in-depth.rst
Comment (anton-trunov) [inline docs/source/scilla-in-depth.rst:1084]: Let's make this sentence look like the rest of them.

File: docs/source/scilla-in-depth.rst
Comment (anton-trunov) [inline docs/source/scilla-in-depth.rst:1089]: We should say something about the cases when the result is `Some ...` and `None`.

File: docs/source/scilla-in-depth.rst
Comment (anton-trunov) [inline docs/source/scilla-in-depth.rst:1091]: Same here. Is `/` supposed to be `.`?

**Review by anton-trunov — CHANGES_REQUESTED** [review]

**Review by anton-trunov — APPROVED** [review]

**Comment by jjcnn** [comment]
Are the changes documented here part of the Scilla update we've just released? If they are, then we should merge this PR, but if they aren't then it might be better to wait until the next release.

**Review by anton-trunov — APPROVED** [review]

**Comment by anton-trunov** [comment]
@jjcnn These builtins are not part of the most recent Scilla release. Let's wait then. (We should really have versioning for docs as we have been discussing for a number of months)

**Comment by jjcnn** [comment]
@anton-trunov : What's the status on these crypto buildins? Are they part of Scilla now, or did we abandon them?

**Comment by anton-trunov** [comment]
Yes, the linked PRs have been merged.

**Pull request merged**
